src/styles/blocks/header.scss

```diff
@@ -0,0 +1,21 @@
+.header,
+.nav__list {
+  display: flex;
+  align-items: center;
+}
+
+.header {
+  font-family: inherit;
+  font-weight: $medium-font-weight;
+  justify-content: space-between;
+  position: relative;
+  height: 60px;
+  width: 100%;
+  background-color: $color-bright;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05) ;
+
+  &__link {
+    display: flex;
+    margin: 10px 0 10px 50px;
```

Comment:
The margin property on the '.header__link' selector could potentially lead to inconsistent layout. If you want to center the element, consider using flexbox properties to center it vertically within the '.header' element instead of using margins.